sqlalchemy-stubs/sql/functions.pyi

```diff
@@ -135,9 +138,15 @@ class _FunctionGenerator:
 func: _FunctionGenerator
 modifier: _FunctionGenerator
 
+class _TypeDescriptor(Protocol[_T_co]):
```

Comment:
we use covariant because we are fine with subclasses of the the type here, right?

Reply:
Covariant is required by `Protocol` in this case